src/game3.py

```diff
@@ -46,40 +46,44 @@ def init_game():
     pygame.display.set_caption('Birdzzz')
 
 def play_game():
-    fixed_events = ["R", "D", "R", "D", "R", "D", "R", "D", "R", "D", "R", "D"]
+    L = pygame.USEREVENT
+    R = pygame.USEREVENT + 1
+    U = pygame.USEREVENT + 2
+    D = pygame.USEREVENT + 3
+
+    fixed_events = [R, D, R, D, R, D, R, D, R, D, R, D]
+    delay = 200
+
+    for event in fixed_events:
```

Comment:
It might be confusing to use `event` twice in this function for two different types of things. What do you think @iuliatamas?